packages/dbos-cloud/cli.ts

```diff
@@ -132,8 +132,9 @@ applicationCommands
   .command('delete')
   .description('Delete this application')
   .argument('[string]', 'application name')
-  .action(async (appName?: string) => {
-    const exitCode = await deleteApp(DBOSCloudHost, appName);
+  .option('--dropdb', 'Drop application database')
```

Comment:
Shouldn't we give a default value of `false` here?

Reply:
This is a [boolean option](https://tsmx.net/commander-options/#Boolean_options). If you don't specify this option, it's by default false. Only when calling the program with `--dropdb` it is set to true.